docs/source/overview/what-is-paraglider.rst

```diff
@@ -0,0 +1,30 @@
+.. _whatisparaglider:
+
+What is Paraglider?
+-------------------
+
+Paraglider is a control plane for cloud networking resources designed to simplify the tenant networking experience. 
+The Paraglider Controller exposes the Paraglider API to tenants and uses public cloud APIs to manage the tenant's cloud network.
+
+---
```

Comment:
Were these `---` what you wanted? It looks like this
<img width="2032" alt="Screenshot 2024-05-01 at 15 48 49" src="https://github.com/paraglider-project/paraglider/assets/12043327/51878d25-0c0f-4eed-ab2b-e2f5e02cc19a">

Reply:
Yeah - I just thought it looked like a wall of text without some way to break it up. Open to other ideas.